package.json

```diff
@@ -49,7 +49,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "geckodriver": "^1.22.1",
     "glob": "^7.1.6",
-    "htmlhint": "^0.14.2",
+    "htmlhint": "^0.16.0",
```

Comment:
This and the package-lock.json changes can be rebased out now

Reply:
Yep. Think we can land this afterwards